tests/parametric/test_tracer_flare.py

```diff
@@ -100,10 +100,12 @@ def trigger_tracer_flare_and_wait(test_agent, task_overrides: Dict[str, Any]) ->
     return tracer_flare
 
 
-def assert_valid_zip(content):
-    flare_file = zipfile.ZipFile(BytesIO(b64decode(content)))
-    assert flare_file.testzip() is None, "tracer_file zip must not contain errors"
-    assert flare_file.namelist(), "tracer_file zip must contain at least one entry"
+# TODO: Update to support Gzip.
+def assert_valid_zip(content, test_library):
+    if test_library.lang != "nodejs":  # Node sends a text file right now.
```

Comment:
instead of using this condition, could we decorate the  **test_tracer_flare** method with the missing_feature annotation?

Reply:
+1, having a `if` inside a test make it library-dependant, and you won't have any usefull information in the feature parity dashbaord

Reply:
I've updated it. Please review again. 😄